db/migrate/20241027052236_change_duration_type_in_courses.rb

```diff
@@ -0,0 +1,10 @@
+class ChangeDurationTypeInCourses < ActiveRecord::Migration[6.0]
+  def up
+    # Change the column type to interval with specific casting
+    change_column :courses, :duration, :interval, using: 'duration * interval \'1 minute\''
```

Comment:
why did you converting data since you don't have any?

Reply:
Actually apu, I already had stored some data in the database as integers. To convert them into `:interval` type I used it. Currently I don't need this as I have cleared  my database. Removed it.